collector/thermal_zone_linux.go

```diff
@@ -70,6 +70,10 @@ func NewThermalZoneCollector(logger log.Logger) (Collector, error) {
 func (c *thermalZoneCollector) Update(ch chan<- prometheus.Metric) error {
 	thermalZones, err := c.fs.ClassThermalZoneStats()
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) || errors.Is(err, os.ErrPermission) || errors.Is(err, os.ErrInvalid) {
```

Comment:
Misses import. But CI already told you hah

Reply:
Hrm but not sure about ErrPermission. I feel in that case it might make sense to fail?

Reply:
We do `ErrPermission` as "not found" as a standard in other collectors.

Reply:
Ah right, kk